pytmx/pytmx.py

```diff
@@ -1271,9 +1271,6 @@ def parse_xml(self, node):
         :return: self
         """
         self._set_properties(node)
-        self.name = node.get('name', None)
```

Comment:
why are these removed?

Reply:
These attributes get set in the self._set_properties(node) call.  That method, in TiledElement, correctly casts the values for these types in the _cast_and_set_attributes_from_node_items method.